### PR TITLE
Installation guide refactoring

### PR DESCRIPTION
**Pull request by palash23jain**
added readMe file, installation details and some refactoring

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -2,31 +2,40 @@
...
   flex-grow: 1;
   display: flex;
   flex-direction: column;
   justify-content: center;
   padding-top: var(--spacing4);
 }
+
+/* .pre-container{
+  max-width: 70rem;
+} */
```
Comment (pritam-kr) [inline style.css]: remove commented code

**Review by pritam-kr — APPROVED** [review]
Good to go...You can merge it.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,51 @@
...
+
+
+## Description
+
+- This is a modular and customizable framework which will save your time in designing your UI.
+- This library is just made on html and css.
+
+## Demo
+
+Image or gif
```
Comment (saifudinbadani) [inline README.md]: add a image later on, otherwise looks good to merge.

**Review by saifudinbadani — APPROVED** [review]

**Review by BelideNikhil — APPROVED** [review]
Good to go

**Review by krate27 — COMMENTED** [review]
You can use a good css formatter to beautify code and indentations. Apart from that looks good, also remove comments not needed. Good to merge

**Pull request merged**